commands/new_function.go

```diff
@@ -85,6 +85,8 @@ the "Dockerfile" lang type in your YAML file.
 		fmt.Printf("Folder: %s created.\n", functionName)
 	}
 
+	createORUpdateGitignore()
```

Comment:
Probably could be: `updateGitignore` - add a comment that it creates it if non-existant

Reply:
I have updated the function name.